.azure-pipelines/install-tools-template.yml

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+steps:
+    - task: UseDotNet@2
+      displayName: 'Use .NET Core SDK 2.x'
+      inputs:
+        debugMode: false
+        version: 2.x
+
+    - task: UseDotNet@2
```

Comment:
Why do we need 2 versions of .NET Core?

Reply:
Some tools depend on the older version.

Reply:
Some tools used in the pipeline depend on netcore2.0. Not the modules.